README.md

```diff
@@ -30,3 +30,11 @@ To run tests in your docker container, you will need to override the environment
 ```
 docker-compose exec -e RAILS_ENV=test app rails test
 ```
+
+## Getting Started
+
+Once your server is running you can hit the api! 
+ex. 'api/v3/public/cards/sure_gamble'
```

Comment:
How about changing this to a full url starting with http://localhost:3000/ so this will be a clickable link?